tests/common/reboot.py

```diff
@@ -223,12 +224,28 @@ def check_reboot_cause(dut, reboot_cause_expected):
     logging.debug("dut {} last reboot-cause {}".format(dut.hostname, reboot_cause_got))
     return reboot_cause_got == reboot_cause_expected
 
-def sync_reboot_history_queue_with_dut(dut):
+
+def sync_reboot_history_queue_with_dut(dut, latest_reboot_type):
     """
     @summary: Sync DUT and internal history queues
     @param dut: The AnsibleHost object of DUT.
+    @param latest_reboot_type: most recent reboot type 
```

Comment:
can you remove latest_reboot_type as input ? since we don't need this any more. same as above.